### PR TITLE
[PEA] CTW make targets

### PR DESCRIPTION
**Pull request by caojoshua**
* `make` builds CTW
* `make run-ctw` runs CTW
* `make run-ctw-no-inline` runs CTW without inlining

### CODE REVIEW

File: PEA/Makefile [hunk 1]
```diff
@@ -1,11 +1,14 @@
 .DEFAULT_GOAL := all
 
-java_path := $(abspath ../build/linux-x86_64-server-fastdebug/images/jdk)
-export JAVA_HOME := ${java_path}
+JAVA_PATH := $(abspath ../build/linux-x86_64-server-fastdebug/images/jdk)
+export JAVA_HOME := ${JAVA_PATH}
 export PATH := ${JAVA_HOME}/bin:$(PATH)
```
Comment (navyxliu) [inline PEA/Makefile:5]: We should avoid from using 'environment variable' here. 

In Makefile, we can always define gmake variables.  I understand that we have to define JAVA_HOME and PATH for auto.sh, we can do that like you did for ctw.sh with leading env JAVA_OPTIONS.

File: PEA/Makefile [hunk 2]
```diff
@@ -15,6 +18,15 @@ set_jdk:
 
 all: set_jdk
 	javac *.java
+	cd ${CTW_DIR}; make
 
 run: set_jdk
 	./auto.sh
+
+run-ctw: set_jdk
+	cd ${CTW_DIR}/dist; \
```
Comment (navyxliu) [inline PEA/Makefile:27]: Should we use pushd/popd here? that will automatically resume the working directory.

Comment (caojoshua) [reply PEA/Makefile:27]: I think it does not matter here. My understanding for Make is that bash commands are run in a subshell. After we `cd`, we are back to the original working directory.

File: PEA/Makefile (hunk 1 above)
Comment (caojoshua) [reply PEA/Makefile:5]: Sorry, don't quite understand. I just moved the logic from auto.sh into Make. I don't think these environment variables are bad for CTW.

**Review by navyxliu — APPROVED** [review]
LGTM. 
one pending thing is that we can't automatically build ctw. 
I will send a PR tmr to fix that. let's merge your work first.

**Pull request merged**